rdmd_test.d

```diff
@@ -310,6 +310,16 @@ void runTests()
         assert(res.status == -SIGSEGV, format("%s", res));
     }
 
+    /* https://issues.dlang.org/show_bug.cgi?id=11997- rdmd should search it's
+    binary path for the compiler */
+    string localDMD = buildPath(tempDir(), baseName(compiler));
+    std.file.write(localDMD, "empty shell");
+    scope(exit) std.file.remove(localDMD);
+
+    res = execute(rdmdApp ~ [modelSwitch, "--force", "--chatty", "--eval=writeln(`Compiler found.`);"]);
+    assert(res.status == 1, res.output);
+    assert(res.output.canFind(`spawn ["` ~ localDMD ~ `",`));
```

Comment:
Oh, heh, seems pointless.  I meant it would be nice if we could actually compile with the adjacent dmd.  If we're not going that far, I actually like that it fails, because the first assert will trigger if `rdmd` pulls a dmd from `PATH`, because it will unexpectedly compile! :) No need to even check the path in the `spawn ["dmd"]` invocation then.